client/card-readers/settings/sections/test/index.test.js

```diff
@@ -71,7 +71,7 @@ describe( 'Card Reader Contact Details section', () => {
 		const email = screen.getByLabelText( 'Support email' );
 		expect( email ).toBeInTheDocument();
 
-		const phone = screen.getByLabelText( 'Support phone number' );
+		const phone = screen.getByLabelText( 'Mobile phone number' );
```

Comment:
Is this change due to the component usage (I guess a hardcoded value there)?

Reply:
Yeah, the label `Mobile phone number` is hardcoded inside the component, so I can't pass one...